android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java

```diff
@@ -254,10 +254,10 @@ public void receiverGetStats(String receiverId, Promise promise) {
     }
 
     public void senderGetStats(String senderId, Promise promise) {
-        RtpReceiver targetSender = null;
-        for (RtpReceiver r : peerConnection.getReceivers()) {
-            if (r.id().equals(senderId)) {
-                targetSender = r;
+        RtpSender targetSender = null;
+        for (RtpSender s : this.peerConnection.getSenders()) {
```

Comment:
one question: here you added `this.` -- Do we need to change that in `receiverGetStats()` too? or was the addition of `this` somehow not necessary?

Reply:
`this` keyword is used to refer to the current instance of the class in which it appears. But it is not mandatory, it is just to make it more explicit.